WordPress/wordpress-playground
# Make kitchen sink extension bundle the default

Many reported issues are caused by Playground defaulting to the "light" PHP build. It saves 6MB of initial downloads, but it confuses new users with issues like [broken image editing](https://github.com/WordPress/wordpress-playground/issues/1190).

This PR changes the PHP bundle loaded by default to the "kitchen sink" one that ships with PHP extensions like finfo, GD, libxml. The change affects all APIs: Query, Blueprints, JS.

I don't like increasing the bundle size, but the `light` one is still an option for apps requiring less transfer and this should reduce the number of confused users and error reports.

## Related issues

* https://wordpress.slack.com/archives/C06Q5DCKZ3L/p1712040143578229?thread_ts=1711960560.447319&cid=C06Q5DCKZ3L
* https://github.com/WordPress/wordpress-playground/issues/1190

 ## Testing instructions

* Confirm the e2e tests passed.
* Go to local Playground
* Visit "/phpinfo.php" in Playground, confirm libxml is enabled (the
  output should contain --enable-xmlreader)
* Open Playground configuration modal, uncheck the additional PHP
  extensions, confirm your choice
* Confirm phpinfo.php now reports libxmls is disabled
* Interact with that modal again, turn on the PHP extensions, confirm
  libxml is loaded again

CC @flexseth @dmsnell @bgrgicak @brandonpayton
